src/main/java/taxi/dao/DriverDaoImpl.java

```diff
@@ -72,13 +74,16 @@ public List<Driver> getAll() {
     public Driver update(Driver driver) {
         String query = "UPDATE drivers "
                 + "SET name = ?, license_number = ? "
+                + "login = ?, password = ?"
                 + "WHERE id = ? AND is_deleted = FALSE";
         try (Connection connection = ConnectionUtil.getConnection();
                 PreparedStatement statement
                         = connection.prepareStatement(query)) {
             statement.setString(1, driver.getName());
             statement.setString(2, driver.getLicenseNumber());
-            statement.setLong(3, driver.getId());
+            statement.setString(1, driver.getLogin());
+            statement.setString(2, driver.getPassword());
+            statement.setLong(5, driver.getId());
```

Comment:
It looks odd, you set the name and  the login in the same position and license number with the password